pkg/cloudprovider/vsphere/loadbalancer/config/config_test.go

```diff
@@ -0,0 +1,103 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func TestReadConfig(t *testing.T) {
+	s1 := `
+[LoadBalancer]
+ipPoolName = pool1
+size = MEDIUM
+lbServiceId = 4711
+logicalRouterId = 1234
+
+[LoadBalancerClass "public"]
```

Comment:
The key name has to be "public" or "private" here? What's the reason we can't make it arbitrary?

Reply:
That's just a test to check whether a given config in a config file  correctly reaches the internal data structures. So we have to use a dedicated (artificial) value.